examples/wordcount/docker/wordcount/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6
+ADD . /code
+
+RUN apt-get update
+RUN echo "/usr/local/lib" >> /etc/ld.so.conf
+RUN git clone https://github.com/edenhill/librdkafka.git /tmp/librdkafka
+RUN ls /tmp/ && cd /tmp/librdkafka && ./configure && make && make install && ldconfig
+
+WORKDIR /code/examples/wordcount/
+RUN pip --version
+#RUN pip install -e git+https://github.com/confluentinc/confluent-kafka-python.git#egg=confluent-kafka
```

Comment:
`confluent_kafka` is a dependency in your setup.py, so with `RUN pip install -e ../../` it should be pulled in...Can you find out if it doesn't get installed (`RUN pip freeze`) or if there is a different problem? you can build the service with `docker-compose build wordcount`